spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1199,77 +1156,72 @@ app register --name log --type sink --uri maven://org.springframework.cloud.stre
 
 stream create --name foostream --definition "time | log"
 
-stream deploy --name foostream --properties "app.*.spring.cloud.stream.bindings.applicationMetrics.destination=metrics,deployer.*.count=2"
+stream deploy --name foostream --properties "app.*.spring.cloud.stream.bindings.applicationMetrics.destination=metrics"
 ----
 
-The Metrics Collector exposes aggregated metrics under the HTTP endpoint
-`/collector/metrics` in JSON format. The Data Flow server accesses this
-endpoint in two distinct ways. The first is by exposing a `/metrics/streams`
-HTTP endpoint that acts as a proxy to the Metrics Collector endpoint. This
-is accessed by the UI when overlaying message rates on the Flow diagrams for
-each stream. It is also accessed to enrich the Data Flow `/runtime/apps`
-endpoint that is exposed in the UI in the `Runtime` tab and in the shell
+The Metrics Collector exposes aggregated metrics under the HTTP endpoint `/collector/metrics` in JSON format.
+The Data Flow server accesses this endpoint in two distinct ways.
+The first is by exposing a `/metrics/streams` HTTP endpoint that acts as a proxy to the Metrics Collector endpoint.
+This is accessed by the UI when overlaying message rates on the Flow diagrams for each stream.
```

Comment:
"Flow diagrams for each stream" -> SCDF's Flo and it's visual representation of streaming pipelines.